src/ui/domemittermixin.js

```diff
@@ -254,6 +254,15 @@ function getNodeUID( node ) {
 	return node[ 'data-ck-expando' ] || ( node[ 'data-ck-expando' ] = utils.uid() );
 }
 
+// Checks if given node is native DOM Node.
+//
+// @private
+// @param {Node} node
+// @return {Boolean} True when native DOM Node.
+function isDomNode( node ) {
+	return node && node.nodeType && node.nodeName;
```

Comment:
I think it unlikely to happen but OK, I'll add `addEventListener` check. 

> BTW. Are you sure that all kinds of DOM nodes have the nodeName defined?

[Yes](https://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-1841493061).